text/checklist.tex

```diff
@@ -120,7 +120,10 @@
 
 Refer to Section~\ref{sec:A1GTRM} Aspect 1: Granularity, Timespan and Reported Measurement for more information about the Level 1 power submission.
 
-For Level 1, the only subsystem included in the power measurement is the compute-node subsystem. The compute-node subsystem is the set of compute nodes. Measure the greater of $ \frac{1}{64} $ of the compute-node subsystem or 1kW of power.
+For Level 1, the only subsystem included in the power measurement is the compute-node subsystem. 
+The compute-node subsystem is the set of compute nodes. 
+Measure a minimum of 2kw of power, 
+10% of the system, or 12  nodes whichever is largest, or the whole machine if smaller than the largest requirement.
```

Comment:
The % sign is a comment character in LaTeX.  The `10%` needs to be changed to `10\%` to render correctly.